File: 06-LetterCapitalize/6.hs
--Capitalize each word in a sentence 

import Data.Char

letterCapitalize :: String -> String
letterCapitalize str = unwords . map (\(x:xs) -> (toUpper x):xs) . words $ str

main = do
  print $ letterCapitalize "hello world" == "Hello World"
  print $ letterCapitalize "this is a test"

File: 06-LetterCapitalize/6.js
// Capitalize each word in a sentence 
function LetterCapitalize(str) { 
  return str.split(' ').map(function(str){
    return str[0].toUpperCase() + str.slice(1)
  }).join(' ')
}

console.log(LetterCapitalize("hello world") == "Hello World")
console.log(LetterCapitalize("this is A test case s s"))